cpg_workflows/stages/large_cohort.py

```diff
@@ -1058,7 +1076,7 @@ def queue_jobs(self, cohort: Cohort, inputs: StageInput) -> StageOutput | None:
                 query_command(
                     browser_vcf_release,
                     browser_vcf_release.run_browser_vcf_data_download.__name__,
-                    freq_ht_path,
+                    prejob.ofile,
```

Comment:
Gut feeling is that this won't work. 

- `ofile` isn't a protected word/value, `job.anything` is just a Hail shorthand for 'a local file path which will be copied between jobs'. In this instance because prejob isn't explicitly written to during the prejob command, it doesn't exist.
- these output files are only single files, so HT/MTs can't be passed this way
- a bash job can't return anything, only PythonJobs can, so you can't get a string result from a job

Suggested restructure https://github.com/populationgenomics/production-pipelines/pull/1330